spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/AccessLevel.java

```diff
@@ -31,7 +31,7 @@
 	/**
 	 * Restricted access to a limited set of endpoints.
 	 */
-	RESTRICTED("", "health", "info"),
+	RESTRICTED("", "health"),
```

Comment:
We do not need to alter the behavior of the CloudFoundry actuator endpoints as they are secured via a different mechanism.